packages/cmf/src/cmfModule.merge.js

```diff
@@ -0,0 +1,122 @@
+import { fork } from 'redux-saga/effects';
+import { assertValueTypeOf } from './assert';
+import { noOp } from 'reassemble';
+
+function mergeObjects(obj1, obj2) {
+	if (!obj2 && obj1) {
+		return obj1;
+	}
+	if (!obj1 && obj2) {
+		return obj2;
+	}
+	return Object.keys(obj2).reduce((acc, key) => {
```

Comment:
If obj1 AND obj2 are undefined, you'll execute that and have an error.
Please change that to
```javascript
if (!obj2) {
	return obj1;
}
if (!obj1) {
	return obj2;
}
return Object/keys(obj2).reduce(....); // here obj1 and obj2 are defined
```